lib/puppet/vendor/semantic/lib/semantic.rb

```diff
@@ -0,0 +1,5 @@
+require 'puppet/vendor/semantic_puppet/lib/semantic_puppet'
```

Comment:
When we look at fixing the Rakefile more sanely / our impacted modules from https://gist.github.com/Iristyle/8e26fe5c59590fe475f6cc2a52f2a5bc#file-author-separated-reduced-list-txt-L183-L193 - I wonder if its enough to just `require 'puppet'` to get the desired `Rakefile` behavior?